cluser_validation_checklist.md

```diff
@@ -0,0 +1,87 @@
+# Cluster Validation Checklist
+## Check general cluster status
+
+ - [ ] Review installation log
+   Check for any glaring issues, cluster admin credentials can be pulled from here as well.
+   Install logs can be found on the install node, inside of the install directory
+   ```shell
+   $ cat <install_dir>/.openshift_install.log
+   ```
+
+ - [ ] Verify that image pull source is what is expected
+   Check that the source of your openshift images is as expected.
+   As we are not in a disconnected environment quay.io and the internal cluster registry.
+   ```shell
+   $ oc adm node-logs <node_name> -u crio
+   ```
+
+ - [ ] Verify that cluster version matches what is expected
+   We'll be running the latest generally available version of OpenShift.
+   At the time of writing that is 4.10.
+   ```shell
+   $ oc get cluserversion
+   ```
+
+ - [ ] Verify that the cluster is on the correct update channel
+   Currently we're looking at the stable update channel for 4.10
+   ```shell
+   $ oc get cluserversion -o jsonpath='{.items[0].spec}{"\n"}'
+   ```
+
+ - [ ] Check for any available cluster updates
+   In the event that a new release has been made available during install check we are on the latest version.
+   ```shell
+   $ oc adm upgrade
+   ```
+ 
+## Verify operator availability
+
+ - [ ] Verify that expected Operators are available
+   We need to generate an initial list of operators that each cluster requires.
```

Comment:
which file in the repo will contain this list?

Reply:
There is no single file in the repository that contains this information. There will be multiple files in the repository that are used for installing operators, and we can query the list of installed operators via the API (or by visually inspecting the web UI).